builder-support/debian/authoritative/debian-buster/rules

```diff
@@ -1,92 +1,68 @@
 #!/usr/bin/make -f
-include /usr/share/dpkg/architecture.mk
-include /usr/share/dpkg/pkg-info.mk
-
-# (Re-)Enable warnings
-CXXFLAGS += -Wall
 
 # Turn on all hardening flags, as we're a networked daemon.
 export DEB_BUILD_MAINT_OPTIONS = hardening=+all
+# see EXAMPLES in dpkg-buildflags(1) and read /usr/share/dpkg/*
+DPKG_EXPORT_BUILDFLAGS = 1
+include /usr/share/dpkg/default.mk
 
-# Backends
-backends := bind ldap pipe gmysql godbc gpgsql gsqlite3 geoip lmdb lua2 remote random tinydns
+# Vendor and version
+CXXFLAGS += -DPACKAGEVERSION='"$(DEB_VERSION).$(DEB_VENDOR)"'
 
-# Only enable systemd integration on Linux operating systems
-ifeq ($(DEB_HOST_ARCH_OS),linux)
-CONFIGURE_ARGS += --enable-systemd --with-systemd=/lib/systemd/system
-DH_ARGS += --with systemd
-else
-CONFIGURE_ARGS += --disable-systemd
-endif
+# Avoid test failure in docker.
+export PDNS_TEST_NO_IPV6=1
+
+# Backends
+backends := bind ldap lmdb lua2 pipe gmysql godbc gpgsql gsqlite3 geoip remote random tinydns
 
-# Only disable luajit on arm64
-ifneq ($(DEB_HOST_ARCH),arm64)
-CONFIGURE_ARGS += --with-lua=luajit
-else
-CONFIGURE_ARGS += --with-lua=lua5.3
-endif
 
-# Use new build system
 %:
-	dh $@ \
-	  --with autoreconf \
-	  $(DH_ARGS)
+	dh $@
+
+override_dh_auto_clean:
+	dh_auto_clean
+	rm -f dnslabeltext.cc
 
 override_dh_auto_configure:
-	test -f pdns/dnslabeltext.cc && mv pdns/dnslabeltext.cc debian/dnslabeltext.cc.moved || true
 	dh_auto_configure -- \
 		--sysconfdir=/etc/powerdns \
+		--enable-systemd --with-systemd=/lib/systemd/system \
 		--with-dynmodules="$(backends)" \
 		--with-modules="" \
+		--enable-ixfrdist \
 		--enable-tools \
-		--with-libsodium \
-		--with-lua=luajit \
 		--with-protobuf \
 		--enable-unit-tests \
-		--enable-experimental-pkcs11 \
 		--enable-lua-records \
-		--enable-ixfrdist \
+		--enable-reproducible \
```

Comment:
we had this disabled in our packages before, so we knew the exact build info.

Reply:
fixed